src/camera.js

```diff
@@ -582,7 +639,8 @@ vgl.camera = function () {
   ////////////////////////////////////////////////////////////////////////////
   this.computeViewMatrix = function () {
     if (m_computeModelViewMatrixTime.getMTime() < this.getMTime()) {
-      mat4.lookAt(m_viewMatrix, m_position, m_focalPoint, m_viewUp);
+      var position = m_position, focalPoint = m_focalPoint;
```

Comment:
Why make new variables here?  Isn't this just a new reference to the same object?